app/src/androidTest/java/app/cybrid/demoapp/ui/kyc/IdentityVerificationActivityTest.kt

```diff
@@ -39,13 +36,14 @@ class IdentityVerificationActivityTest {
 
     private lateinit var resources: Resources
 
-    @Before
+    //@Before
     fun setup() {
 
         resources = InstrumentationRegistry.getInstrumentation().context.resources
     }
 
-    @Test
+    //@Ignore("Disabled until KYC Bearer Token in ready")
+    //@Test
```

Comment:
??

Reply:
Do you remeber that I gonna turn off the e2e testing for Indentity Verification Component!

Reply:
Ok. But we should add a TODO comment in the code stating that's what's happening so we know to go back and resolve it.